domains/certificates/Query.API/API/Protos/meteringpoint.proto

```diff
@@ -35,4 +35,5 @@ message MeteringPoint {
   string consumer_Cvr = 18;
   string municipality_code = 19;
   string city_sub_division_name = 20;
+  string metering_grid_area_id = 21;
```

Comment:
Should this be an array to support "A meteringpoint can in some cases have two GridAreaIds if it is part of an exchange between two grid-areas." ?